src/widgets_controller.tsx

```diff
@@ -37,6 +37,7 @@ export class WidgetsController {
         plans,
         transitionDelay,
         hideIfNotEligible,
+        monochrome,
```

Comment:
```suggestion
        monochrome = true,
```
You should initialize `monochrome` here, otherwise monochrome will be undefined if not set and active item will be underlined in red and Alma logo in blue
![Capture d’écran du 2022-05-10 15-03-53](https://user-images.githubusercontent.com/104012464/167795533-259aa8c6-5671-4237-88b3-100e6294674a.png)